Libraries/NativeAnimation/RCTAnimatedValueChangeEvent.h

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import <React/RCTEventDispatcher.h>
+
+@interface RCTAnimatedValueChangeEvent : NSObject <RCTEvent>
+
+- (instancetype)initWithNodeTag:(NSNumber *)nodeTag
```

Comment:
If it is not a `viewTag`, should we don't implement `viewTag` property at all and mark `viewTag` as optional in the protocol.

Reply:
This should be possible, we only use viewTag to generate an id for the event based on viewTag + eventName + coalescingKey. We could make view tag optional for events that are not related to views and only use coalescingKey to identify events. (https://github.com/facebook/react-native/blob/master/React/Base/RCTEventDispatcher.m#L34)